robot_sdk/wrappers/68_wrapper.py

```
@@ -13,54 +18,63 @@ def create_pose_from_point(point):
         '''
         Function to enable hand drag
         ''' 
-        print("Can't enable hand drag in Simulation")
-    
-    def move_to_pose(self,pose,tool_speed : int = 100):
+        if switch is True:
+            return self.client.enter_teach_mode()
+        elif switch is False:
+            return self.client.end_teach_mode()
+
+    def move_to_pose(self,pose,tool_vel : float= 0.20, tool_acc : float = 0.01):
```

Ruff (0.12.2)

Unused method argument: `tool_vel`

(ARG002)